docs/development.md

```diff
@@ -504,3 +504,50 @@ Issue is a duplicate of other already existing issue.
 #### [invalid](https://github.com/kedgeproject/kedge/labels/invalid)
 
 #### [wontfix](https://github.com/kedgeproject/kedge/labels/wontfix)
+
+
+## Code instructions
+
+### Things to take care of when adding new type:
+
+- Creating type struct
+
+  - Make a struct type named `ControllerSpecMod` and add it to [types.go](https://github.com/kedgeproject/kedge/blob/cd5297cfdbd2f1daa510824d49c9d3d649ffc0b8/pkg/spec/types.go).
```

Comment:
Maybe mentioning that it should contain ObjectMeta?

Reply:
ObjectMeta is already present in `ControllerFields`, so when doing controller types we don't need it additionally!